net.sourceforge.wxEDID.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.sourceforge.wxEDID</id>
+  <name>wxEDID</name>
+  <summary>Edit edid</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">net.sourceforge.wxEDID.desktop</launchable>
+  <content_rating type="oars-1.1"/>
+  <description>
+    <p>
+      wxEDID is a wxWidgets - based EDID (Extended Display Identification Data) editor.
+    </p>
+    <p>
+      This is an early stage of development, allowing to modify the base EDID v1.3+ structure and CEA/CTA-861-G (as first extension block).
+    </p>
+    <p>
+      Besides normal editor functionality, the app has been equipped with a DTD constructor, which aims to ease timings selection/editing. It's also possible to export and import EDID data to/from text files (hex ASCII format) and also to export the structures as a human-readable text.
+    </p>
+  </description>
+  <url type="homepage">https://sourceforge.net/projects/wxedid/</url>
+  <releases>
+    <release version="32.0.0.465" date="2020-12-08"/>
+  </releases>
+</component>
```

Comment:
missing required `screenshots`

Reply:
Would these hosted on the sourceforge cdn be acceptable?
`https://a.fsdn.com/con/app/proj/wxedid/screenshots/BlockTree-801eb1dc.png/max/max/1`
`https://a.fsdn.com/con/app/proj/wxedid/screenshots/DTD_Ctor-216978b8.png/max/max/1`